libs/libcommon/src/libcommon/processing_graph.py

```diff
@@ -593,8 +633,8 @@ def parse_id(id: str) -> tuple[str, str, Optional[str], Optional[str], str]:
         parts = parts[2:]
         config = None
         split = None
-        if len(parts) > 1:
-            config = parts[1]
-            if len(parts) > 2:
-                split = parts[2]
+        if len(parts) > 0:
```

Comment:
it was wrong? and had no consequences until now?

Reply:
It was wrong and had no consequences since this function was only used on jobs at dataset level in tests

Reply:
OK, thanks for checking!